redpanda/values.yaml

```diff
@@ -156,15 +181,37 @@ listeners:
     - address: 0.0.0.0
       name: default
       port: 8081
+      # Optional external section
       external:
-        enabled: true
+        # enabled: true
+        # Type of external access (options are NodePort and LoadBalancer)
+        # type: NodePort
+        # External port
+        # This listener port will be used for the external port if this is not included
+        port: 30081
       tls:
         # enabled: true
         cert: schema
         requireClientAuth: false
 #
+# External access configuration
+external:
+  # Default external access value for all listeners except RPC
+  # External config doesn't apply to RPC listeners as they are never externally accessible
+  # These values can be overridden by each listener if needed
+  enabled: true
+  # Default external access type (options are NodePort and LoadBalancer)
+  # TODO include IP range for load balancer that support it: https://github.com/redpanda-data/helm-charts/issues/106
+  type: NodePort
+  domain: local
+  # annotations:
+    # For example:
+    # cloud.google.com/load-balancer-type: "Internal"
+    # service.beta.kubernetes.io/aws-load-balancer-type: nlb
```

Comment:
This is the most important change to `values.yaml` as far as this PR is concerned. This is the top-level external section.